File: lc1365.java
import java.util.Arrays;

public class lc1365 {
    public static void main(String[] args) {
//        1365. How Many Numbers Are Smaller Than the Current Number
        int nums[]={8,1,2,2,3};
        int ans[]=new int[nums.length];
        for (int i=0;i<nums.length;i++){
            int c=0;
            for (int j=0;j< nums.length;j++){
                if (i!=j){
                    if (nums[j]<nums[i]){
                        c++;
                    }
                }
            }
            ans[i]=c;
        }
        System.out.println(Arrays.toString(ans));
    }
}

File: lc1365.py
from typing import List

class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        ans=[]
        for i in range(len(nums)):
            smaller = [n for n in nums if n < nums[i]]
            ans.append(len(smaller))
        return ans